static/sass/main.scss

```diff
@@ -328,15 +309,8 @@ footer {
   color: var(--warning-color);
 }
 
-// Enable sticky footer when the screen is wide and long enough
+// Add margin-bottom when sticky footer is enabled
```

Comment:
This is fine.  However, I think that you could take it a step further by eliminating this whole block of SASS.   Instead of the margin being added to @content-column, that space would be in the height or margin of a `<div id="footer-space"></div>` element inside `chromedash-footer` as a sibling to the `<footer>` element.  I'm just brainstorming, I haven't tried it myself.

Reply:
This is a great idea! As you suggested, I added a `<div>` in the footer component as a spacer and removed the css in main.scss.